### PR TITLE
Create initializers using const op

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

**Comment by jiafatom** [comment]
Also set SoftMax axis=-1 (previously we only do it inside dense layer, now according to https://keras.io/activations/, we should make it consistent with keras io).

**Comment by jiafatom** [comment]
This change cannot build for old tf2onnx 0.3.2, it works for the latest tf2onnx.
